src/base/message_builder.rs

```diff
@@ -2332,6 +2332,7 @@ impl<Target: Composer> Truncate for TreeCompressor<Target> {
 #[derive(Clone, Copy, Debug)]
 pub enum PushError {
```

Comment:
It might be nice to document these variants to explain what they mean in rustdoc.